en/dev_setup/dev_env_mac.md

```diff
@@ -87,7 +87,7 @@ brew install px4-sim-gazebo
 ## jMAVSim Simulation
 
 To use SITL simulation with jMAVSim you need to install a recent version of Java (e.g. Java 15).
-You can download [Java 15 (or later) from Oracle](https://www.oracle.com/java/technologies/javase-downloads.html#JDK15) or use the [Termium](https://adoptium.net):
+You can download [Java 15 (or later) from Oracle](https://www.oracle.com/java/technologies/javase-downloads.html#JDK15) or use [Termium](https://adoptium.net):
```

Comment:
@julianoes Thoughts on this, or who can confirm it is OK?

Reply:
`brew install --cask temurin` installs something newer than 15 already.

Reply:
If it works, then we can merge it.

Reply:
I used it to run jMAVSim, so can confirm that it works.

Reply:
Thank you very much.